lib/fpm/package/rpm.rb

```diff
@@ -562,8 +562,8 @@ def epoch
 
   def to_s(format=nil)
     if format.nil?
-      return super("NAME-VERSION-ITERATION.DIST.ARCH.TYPE").gsub('DIST', attributes[:rpm_dist]) if attributes[:rpm_dist]
-      return super("NAME-VERSION-ITERATION.ARCH.TYPE")
+      return super("NAME-VERSION-ITERATION.DIST.ARCH.EXTENSION").gsub('DIST', attributes[:rpm_dist]) if attributes[:rpm_dist]
```

Comment:
This is something that looks strange to me. Why we need to perform substitutions on a fixed (ad-hoc) string? Merely, it's same as
```
super("NAME-VERSION-ITERATION.#{attributes[:rpm_dist]}.ARCH.EXTENSION") if attributes[:rpm_dist]
```
except the current substitution is made on a string already processed by ``super`` (which doesn't really matter ATM, as ``super`` does not introduce any ``"DIST"`` substrings). I'd suggest to delegate this gsub to ``FPM::Package#to_s`` for readability.